java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -3945,16 +3945,18 @@ void testGroupByScan() {
               GroupByScanAggregation.min().onColumn(2),
               GroupByScanAggregation.max().onColumn(2),
               GroupByScanAggregation.rank().onColumn(3),
-              GroupByScanAggregation.denseRank().onColumn(3));
+              GroupByScanAggregation.denseRank().onColumn(3),
+              GroupByScanAggregation.percentRank().onColumn(3));
            Table expected = new Table.TestBuilder()
-               .column( "1",  "1",  "1",  "1",  "1",  "1",  "1",  "2",  "2",  "2",  "2")
+               .column(  "1",  "1",  "1",  "1",  "1",  "1",  "1",  "2",  "2",  "2",  "2")
                .column(   0,    1,    3,    3,    5,    5,    5,    5,    5,    5,    5)
                .column(12.0, 14.0, 13.0, 30.0, 17.0, 34.0, null, null, 11.0, null, 21.0)
                .column(   0,    0,    0,    1,    0,    1,    2,    0,    1,    2,    3) // odd why is this not 1 based?
```

Comment:
As a reviewer, I feel obligated to echo this comment's question. Is this comment worthy of concern / removal / fixing?

Reply:
Wheelllllll.....this is typo....

Reply:
Since this test isn't related to the `percent_rank` aggregation, might we file a separate issue to explore this?

Also, what's the typo we're referring to? The space was to align the values in the rows.

Reply:
We meant why the change is just adding one space? If not a typo, then what is the cause of adding such space? Is it auto-formatted?

Reply:
From your latest comment, I can see that the reason here is "to align value in the row". And this answers my question above.

Reply:
I have filed #10237 to investigate the apparently bad counts.